doc/conf.py

```diff
@@ -44,7 +44,6 @@
     'sphinx.ext.autosummary',
     'sphinx.ext.viewcode',
     'sphinx.ext.githubpages',
-    'matplotlib.sphinxext.only_directives',
```

Comment:
Just curious, what was the desired change

Reply:
Nevermind Found the comment https://mail.google.com/mail/u/1/#inbox/FMfcgxvzKkwmXtvtShWhBDkNHGJqCMXw